src/app/cluster/details/external-cluster/external-machine-deployment-list/component.ts

```diff
@@ -31,6 +31,11 @@ import {ExternalMachineDeploymentService} from '@core/services/external-machine-
 import {UpdateExternalClusterMachineDeploymentDialogComponent} from '@app/cluster/details/external-cluster/update-external-cluster-machine-deployment-dialog/component';
 import {HealthStatus} from '@shared/utils/health-status';
 
+enum AKSState {
```

Comment:
This is specific to NodePool can we rename to 
```suggestion
enum AKSNodePoolState {
```